rapid7_vulndb/komand_rapid7_vulndb/util/utils.py

```diff
@@ -0,0 +1,311 @@
+from komand.exceptions import PluginException
+
+from typing import Dict, List, Callable, Tuple
+import requests
+import copy
+import sys
+from urllib.parse import urljoin
+
+
+class R7VDB:
+    """
+    Rapid7 vulnerability database API utilities
+    This class is used by the rapid7_vulndb plugin for
+    extracting https://vdb.rapid7.com/swagger_doc API data
+
+    Two main functions that are used by the plugin are
+        1. paginate_search
+        2. make_content_result
+
+    most of the methods implemented as class methods to avoid initialization
+    """
+
+    search_url: str = "https://vdb-kasf1i23nr1kl2j4.rapid7.com/v1/search"
+    content_url: str = 'https://vdb-kasf1i23nr1kl2j4.rapid7.com/v1/content/'
+    seach_fields: List[str] = ["title", "published_at", "identifier"]
```

Comment:
```suggestion
    search_fields: [str] = ["title", "published_at", "identifier"]
```
No need to specify List as a type hint when you've already done it via shorthand `[str]`